entrypoint.sh

```diff
@@ -67,6 +67,12 @@ check_configured() {
   esac
 }
 
+check_sendmail() {
+	if [[ ${MAIL_DRIVER} == "sendmail" ]]; then
+		sudo postfix start
```

Comment:
sbin should not be in PATH, so if I'd expect the postfix command won't be found

(in the sudoers file you explicitly state the full postfix path)

Reply:
`postifx` is started correctly, but changing this will not hurt at all.